packages/waku/src/lib/vite-rsc/ssr.tsx

```diff
@@ -47,32 +44,57 @@ export async function renderHTML(
 
   // render html
   const bootstrapScriptContent = await loadBootstrapScriptContent();
-  const htmlStream = await renderToReadableStream(<SsrRoot />, {
-    bootstrapScriptContent:
-      getBootstrapPreamble({ rscPath: options?.rscPath || '' }) +
-      bootstrapScriptContent,
-    onError: (e: unknown) => {
-      if (
-        e &&
-        typeof e === 'object' &&
-        'digest' in e &&
-        typeof e.digest === 'string'
-      ) {
-        return e.digest;
-      }
-      console.error('[SSR Error]', captureOwnerStack?.() || '', '\n', e);
-    },
-    ...(options?.nonce ? { nonce: options.nonce } : {}),
-    ...(options?.formState ? { formState: options.formState } : {}),
-  });
-
+  let htmlStream: ReadableStream<Uint8Array>;
+  let status: number | undefined = undefined;
```

Comment:
Just my preference.
```suggestion
  let htmlStream: ReadableStream;
  let status: number | undefined;
```

Or, is specifying `Uint8Array` helpful? If so, let's do it for all in a separate PR.